### ISSUE TITLE
Add Closing Page

### ISSUE THREAD

**Issue by denoire** [denoire/denoire.github.io#21]
### Purpose
Add the images, hyperlinks, and all other necessary media components to each page.

### Details
Placeholders have been implemented in #13, and only require an ```href`` replacement and alt text. A custom reuseable component should be created to house each hyperlink.

### Acceptance Criteria
- [ ] Add the necessary images to ```public/images```
- [ ] Replace placeholder images with final images
- [ ] Include proper alt text for each image
- [ ] Create a custom reuseable hyperlink component
- [ ] Implement all hyperlinks using the custom component
